dotcom-rendering/src/static/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+	"background_color": "#052962",
+	"display": "minimal-ui",
+	"display_override": ["minimal-ui"],
```

Comment:
What does this field do when it's set to the same value as `display`?

Reply:
i'm not 100% sure - chrome complained when it was missing. i dont' think `standalone` is a good fallback though, because minimal UI will give it back/forward and refresh buttons. 
maybe display should be `browser`?